src/manifest.json

```diff
@@ -31,7 +31,14 @@
             "js/background_scripts/message-listener-instance.js"
         ]
     },
+    "icons": {
+        "16": "images/gamepad-icon-16px.png",
+        "32": "images/gamepad-icon-32px.png",
+        "48": "images/gamepad-icon-48px.png",
+        "128": "images/gamepad-icon-128px.png"
+    },
     "browser_action": {
+        "default_icon": "images/gamepad-icon.png",
```

Comment:
Why did you use `./html` for the popup but not `./images` here?  They should be consistent unless there's a reason they need to be different.

Reply:
This still needs to be done apparently.